src/main/resources/swagger/api-docs.yaml

```diff
@@ -694,6 +698,12 @@ definitions:
           Defaults to false.
         type: boolean
         default: false
+      autopause:
+        type: boolean
+        description: Whether autopause feature is enabled for this specific cluster.
```

Comment:
Can you also say: If unset, autopause will be enabled and a system default threshold will be used.

Reply:
should that default be mentioned somewhere, too?

Reply:
We could say "a system default of 30 minutes". If we change the default however, we need to remember to change the Swagger doc. OTOH, if the user cares about the value they should probably specify the value themselves. :)

Reply:
fair enough